gnd/src/main/java/com/google/android/gnd/repository/DataRepository.java

```diff
@@ -130,12 +131,20 @@ private void onProjectLoaded(Project project) {
     cache.setActiveProject(project);
     activeProject.onNext(Persistable.loaded(project));
     localValueStore.setLastActiveProjectId(project.getId());
+    localDataStore.activateProject(project).subscribeOn(Schedulers.io()).subscribe();
+  }
+
+  private Single<List<Project>> loadProjects(User user) {
+    if (isOfflineModeEnabled()) {
```

Comment:
Related to the previous comment about explicit offline checking. 

I believe our approach was to try to connect to remotes by default and if that doesn't succeed, fallback to local stores, without really making this reliant on the user (while still informing them)--in other words, offline enabled by default and by design.

This might have changed though! Just a heads up based on my prior understanding. I defer to @gino-m

Reply:
Please see https://github.com/google/ground-android/pull/246#discussion_r353000972

Reply:
Nope, still the same!